rmf_traffic_ros2/src/rmf_traffic_schedule/ScheduleNode.cpp

```diff
@@ -293,6 +295,16 @@ ScheduleNode::ScheduleNode()
         }
       }
     });
+
+  // Warning for Foxy users to not use rmw_fastrtps_cpp
+  std::string rmw_env_var = rcpputils::get_env_var("RMW_IMPLEMENTATION");
+  if (rmw_env_var.empty() || rmw_env_var == "rmw_fastrtps_cpp")
+  {
+    RCLCPP_WARN(get_logger(), "RMF is known not to work correctly when using "
```

Comment:
```suggestion
    RCLCPP_WARN(get_logger(), "RMF is known to not work correctly when using "
```

Maybe it's a regional dialect difference, but "known not to" sounds a bit off to my ear.